core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -121,6 +121,8 @@ function (string $value): string {
                         )
                     ->end()
                 ->end()
+                ->scalarNode('environment')
```

Comment:
I guess we could set `defaultValue('')` so it's always set.

Reply:
Okay, dass löst dann auch das Problem mit
 `if (System::getContainer()->hasParameter('contao'))`